CONTRIBUTING.md

```diff
@@ -2,51 +2,57 @@
 
 Great!! We are always on the lookout for more OpenEBS hackers. You can get started by reading this [Overview](./contribute/design/README.md)
 
-OpenEBS is innovation in OpenSource. You are welcome to contribute in any way you can and all the help you can provide is very much appreciated. 
+First: if you're unsure or afraid of anything, just ask or submit the issue or pull request anyways. You won't be yelled at for giving your best effort. The worst that can happen is that you'll be politely asked to change something. We appreciate any sort of contributions, and don't want a wall of rules to get in the way of that.
 
-- Raise Issues on either the functionality or documentation
-- Submit Changes to Improve Documentation 
-- Submit Proposals for new Features/Enhancements 
-- Submit Changes to the Source Code
+However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover what we're looking for. By addressing all the points we're looking for, it raises the chances we can quickly merge or address your contributions.
```

Comment:
we're = you  are 
it raises the chances of quickly merging or addressing your contributions.

Reply:
However, for those individuals who want a bit more guidance on the best way to contribute to the project, read on. This document will cover all the points we're looking for in your contributions, raising your chances of quickly merging or addressing your contributions.